geomesa-convert/geomesa-convert-xml/src/main/scala/org/locationtech/geomesa/convert/xml/XMLConverter.scala

```diff
@@ -58,9 +83,11 @@ class XMLConverterFactory extends SimpleFeatureConverterFactory[String] {
     val idBuilder = buildIdBuilder(conf.getString("id-field"))
     // feature path can be any xpath that resolves to a node set (or a single node)
     // it can be absolute, or relative to the root node
-    val featurePath  = if (conf.hasPath("feature-path")) Some(conf.getString("feature-path")) else None
-    val xsd = if (conf.hasPath("xsd")) Some(conf.getString("xsd")) else None
-    new XMLConverter(sft, idBuilder, featurePath.map(xpath.compile), xsd, fields)
+    val featurePath = if (conf.hasPath("feature-path")) Some(conf.getString("feature-path")) else None
+    val xsd         = if (conf.hasPath("xsd")) Some(conf.getString("xsd")) else None
+    val lineMode    = LineMode.getLineMode(conf)
+    val validate    = isValidating(conf)
```

Comment:
I feel like it was doing some validation, even though validate was set to false... i played around with it some originally, but don't remember all the details now.